content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,92 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you’ve just created the reference entity `Brand`, you’ve defined it’s records' attributes and now it’s time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
+
+To add a new brand to the reference entity `Brand`:
+1. Go to your `Brand` entity (you are automatically on the records grid).
+1. Then click on the `Add new` button, choose a label for the record (the code is automatically generated but you can change it if needed).
+
+::: tips
+If you have several records to add, just check `Create another record` and save: you’ll stay in the creation pop-in and you can quickly add as many records as needed.
+:::
+
+## Display records
+To see the records of the reference entity, click on the `Records` tab in the left menu. You will see the records grid, where all the records of the entity are listed.
+
+### The records grid
+The records' grid is quite different than the products' one. Thereafter are the novelties of this grid:
+1. The first line is frozen: it allows you to scroll down the page, while continuing to see the name of the attributes
+1. The first 4 columns are also frozen since these are the system attributes and the status of the completeness. We added an horizontal scroll in order to see the content of all the columns.
```

Comment:
The first 4 columns with the image, label, code and completeness...
Remove "system attributes"